### DISCUSSION TITLE
Farsi Translation

### DISCUSSION THREAD

**Discussion by alisadeghiaghili** [pcottle/learnGitBranching#856] [General]
Hey folks. I can translate it into Farsi (Persian) if you like.

**Comment by pcottle** [comment]
That would be great! It's a fair amount of content to translate so that's my only disclaimer, but I'd appreciate even just a few levels if you have the time.

A lot of languages get started by one person (and are left incompletely translated) and then other contributors add to their work over time

**Comment by eelhak222-tech** [comment]
Lost the password.

Sorry.

 
 

Sent: Saturday, July 10, 2021 at 2:25 PM
